providers/oidc.go

```diff
@@ -14,12 +14,15 @@ import (
 	"github.com/oauth2-proxy/oauth2-proxy/pkg/requests"
 )
 
+const email = "email"
```

Comment:
Could you rename this to maybe `emailClaim` so that it's more obvious what this is supposed to be used for and also so we don't shadow it on line 275?
```suggestion
const emailClaim = "email"
```